app/game/[gameId]/GameManager.tsx

```diff
@@ -2,8 +2,8 @@
 import React from 'react'
 import { createGameCode, startGame } from '../index'
 
-export const GameManager = (props: { gameId: string }) => {
-  const [players, setPlayers] = React.useState<string[]>([]);
+export const GameManager = (props: { gameId: string, initialPlayers: string[] }) => {
```

Comment:
Nit: Can we de-structure the object within the input, so something like: 
```
({ gameId, initialPlayers }: { gameId: string, initialPlayers: string[] })
```
And potentially lift the type to an alias like:
```
interface GameManagerProps {
 gameId: string;
 initialPlayers: string[];
}
```

Reply:
I agree honestly. I think the props probably should have been lifted in the first place... 👀